apps/chat/src/app.module.ts

```diff
@@ -36,7 +36,7 @@ import mysqlConfig from './config/mysql.config';
       inject: [ConfigService],
       useFactory: async (configService: ConfigService) => {
         const chatMongooseOptions: MongooseModuleOptions = {
-          uri: `${configService.get('mongo.host')}:${configService.get('mongo.port')}`,
+          uri: 'mongodb://root:grabb@svc.sel4.cloudtype.app:32138/GrabbmeChat?authSource=admin',
```

Comment:
```suggestion
          uri: configService.get('mongo.chat_uri')
```
같은 식으로, 해당 mongoDB 연경 URI 를 최대한 은닉하는게 좋은 것 같습니다.
configService 쪽에 의존 시키는게 추후 관리하기도 더 편할 것 같아요!

Reply:
[8531a62](https://github.com/GrebbMe/grabbme-be/pull/35/commits/8531a622e20e9a68c6c9ac803736293ba784aec9)

위 커밋에 반영하였습니다!